app/serializers/api/protocol_subscription_serializer.rb

```diff
@@ -33,12 +33,14 @@ def max_still_awardable_euros
     end
 
     def euro_delta
+      return 0 if no_streak_detected
       latest_streak_value = completion[object.latest_streak_value_index]
       return 0 unless latest_streak_value.present?
       object.protocol.calculate_reward([latest_streak_value])
     end
 
     def current_multiplier
+      return 1 if no_streak_detected
       current_completion = completion[object.latest_streak_value_index]
       return 1 unless current_completion.present?
       latest_streak_value = completion[object.latest_streak_value_index][:streak]
```

Comment:
waarom hier geen `current_completion` btw? :p

Reply:
Dat is jouw code. Ik pas het wel aan.